examples/mockups/index.html

```diff
@@ -188,11 +188,11 @@ <h2 class="ui header">How it works</h2>
           
               <div class="ui basic vertical huge segment">
                 <p>
-                  We generate, host and manage your documentation versions for you, outputting both html and pdf formats. 
+                  We generate, host and manage your documentation versions for you, shipping both HTML and PDF. 
                   Whenever you push code, we'll build your docs.
                 </p>
                 <p>
-                  <big>It Just works.</big>
+                  <big>It just works.</big>
```

Comment:
@agjohnson I know you're not a fan of this slogan but I quite like it! Today is my support day and I received zero requests yet. I think for the majority of our users, "it just works". And it's also a sentence we heard a lot during the customer development interviews. What do you think?

Otherwise, we could maybe replace it by another call for action. "Import your docs now" with a button.

Reply:
I don't have strong feelings for "it just works", one way or another, but I'd love to have a bold statement here!
We also talked about the importance of having a slogan for the product.

Either way, adding an action button here seems a good idea.

Reply:
So yeah I'm +1 on a bold statement here. But "it just works" sounds too simplified and vague and a bit handy wavy to me. "It just works" could be applied almost anywhere -- electrical service to my house just works :slightly_smiling_face:. But I *do* like our sentiment behind this statement: users get a lot of value out of minor interaction with us.

I think we can retain the sentiment, and be less vague/generic with this. Clearer language would go a long way here. There were a few suggestions here that I think make sense to explore: focus on what matters: your documentation, spend less time on infrastructure around your documentation, etc. Snappier of course though.

Reply:
`Get your docs online in 5 minutes` or something perhaps? I do agree we should deconstruct what "just works". Meaning it's easy to setup? Doesn't require much maintenance?

Reply:
`Get your docs online in 5 minutes` is a great example -- it's more specific and conveys at least some of what we offer. I do think there is something more specific that we'd eventually want here, probably describing automation or ease of doc management, but this is a good starting point.

Reply:
"Get your docs online in 5 minutes" is nice, but doesn't convey "and then forget that we exist" -- although that's admittedly overly optimistic for some use cases, most of our projects do zero maintenance on their docs once they're online, which is what I somehow wanted to communicate here.

Reply:
In any case, changing to `Get your docs online in 5 minutes` which is less contentious.

Reply:
Yeah, I agree here as well, but at least prefer "Get your docs online" over a "It works". I think the above suggestion on focusing on automation or management is the correct focus for this block long term.